tensorflow/conv2d.py

```diff
@@ -125,68 +161,79 @@ def bench(
     filter_width: int,
     filter_input_channels: int,
     filter_output_channels: int,
+    version: str,
 ):
-    g = tf.Graph()
-    with tf.device("/%s:0" % (_ARGS_DEVICE)) and g.as_default():
-        if _ARGS_DATA_FORMAT == "NHWC":
-            init_tensor = tf.Variable(
+    times = []
+    if int(version[0]) >= 2:
```

Comment:
It could be better to create two separate functions `bench_v1` and `bench_v2`. This function is called in a loop and it could be a good idea to avoid branching noise.

Reply:
Thanks, it was my second option. I will change it :)